README.md

```diff
@@ -5,49 +5,110 @@
 
-`sentryR` is an unofficial R client for [Sentry](https://sentry.io). 
-Currently tested with Plumber APIs.
-
+`sentryR` is an unofficial R SDK for [Sentry](https://sentry.io).
+It includes an error handler for Plumber for uncaught exceptions.
 
 ## Installation
 You can install the current development version of `sentryR` with:
 
 ``` r
-# install.packages("devtools")
-devtools::install_github("ozean12/sentryR")
+# install.packages("remotes")
+remotes::install_github("ozean12/sentryR")
 ```
 
-## Example
+## Using sentryR
 
-Configure your project in [Sentry](https://sentry.io), and set its DSN as an environment variable.
+`configure_sentry` and `capture` are the two core functions of `sentryR`.
+The first sets up an isolated environment with your Sentry project's DSN,
+optionally your app's name, version and the environment it's running in.
+Both `configure_sentry` and any of the `capture_` functions accept 
+additional fields to pass on to Sentry as named lists. 
+`NULL`ifying a field will remove it.
 
-Example Plumber API:
 ```r
-library(plumber)
 library(sentryR)
 
-sentry.config(Sys.getenv('SENTRY_DSN'))
+configure_sentry(dsn = Sys.getenv("SENTRY_DSN"),
+                 app_name = "myapp", app_version = "8.8.8",
+                 environment = Sys.getenv("APP_ENV"),
+                 tags = list(foo = "tag1", bar = "tag2"),
+                 runtime = NULL)
+
+capture(message = "my message", level = "info")
+```
+
+You are encouraged to use the two wrappers around `capture` included:
+`capture_exception`, which handles the error object and then reports the error
+to Sentry, and `capture_message` for transmitting messages.
+Refer to the [Sentry docs](https://docs.sentry.io/development/sdk-dev/event-payloads/)
+for a full list of available fields.
+
+By default `sentryR` will send the following fields to Sentry:
+```r
+list(
+  logger = "R",
+  platform = "R", # Sentry will ignore this for now
+  sdk = list(
+    name = "SentryR",
+    version = ...
+  ),
+  contexts = list(
+    os = list(
+      name = ...,
+      version = ...,
+      kernel_version = ...
+    ),
+    runtime = list(
+      version = ...,
+      type = "runtime",
+      name = "R",
+      build = ...
+    )
+  ),
+  timestamp = ...,
+  event_id = ...
+  modules = (installed packages)
+)
+```
+
+`capture_exception` further adds the `exception` field to the payload.
+
 
-defaultErrorHandler <- plumber:::defaultErrorHandler()
+## Example with Plumber
 
-errorHandler <- function(req, res, err) {
-  sentry.captureException(err, req)
-  defaultErrorHandler(req, res, err)
-}
+In a Plumber API, besides the initial configuration for Sentry, 
+you'll also have to set the error handler to `sentry_error_handler`
+
+```r
+library(plumber)
+library(sentryR)
+
+configure_sentry(dsn = Sys.getenv('SENTRY_DSN'), 
+                 app_name = "myapp", app_version = "1.0.0")
 
 r <- plumb("R/api.R")
-r$setErrorHandler(errorHandler)
+r$setErrorHandler(sentry_error_handler)
```

Comment:
Should this explain how to wrap the default error handler (or your own) instead?